Documentation/dev/libvirt-howto.md

```diff
@@ -83,7 +83,7 @@ range used by your cluster nodes.  If you're using the default subnet of
 `192.168.124.0/24`, something along these lines should work:
 
 ```
-iptables -I INPUT -p tcp -s 192.168.124.0/24 -d 192.168.124.1 --dport 16509 \
+iptables -I INPUT -p tcp -s 192.168.124.0/24 -d 192.168.122.1 --dport 16509 \
```

Comment:
This should almost certainly match the example from a few lines up (e.g. both be `124` or both be `122`).  It should also probably match the example [over here][1].

[1]: https://github.com/openshift/installer/blob/00862d41c0d5cdb0f92416161651e47ce2481b40/examples/libvirt.yaml#L17

Reply:
You may also want to mention:

```console
$ ip -4 address show dev virbr0
4: virbr0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN group default qlen 1000
    inet 192.168.122.1/24 brd 192.168.122.255 scope global virbr0
       valid_lft forever preferred_lft forever
```

as a way to extract that IP (or maybe there's a way to extract it with `virsh`?).

Reply:
> This should almost certainly match the example from a few lines up (e.g. both be `124` or both be `122`). It should also probably match the example [over here](https://github.com/openshift/installer/blob/00862d41c0d5cdb0f92416161651e47ce2481b40/examples/libvirt.yaml#L17).

I see, the issue here is ambiguity. We're suggesting the *124* subnet as a HOWTO-wide default for the testing network, but the *122* is the subnet of the libvirt network named *default*.

Reply:
Testing this now, I have:

```
$ ip -4 addr show tt0
68: tt0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    inet 192.168.124.1/24 brd 192.168.124.255 scope global tt0
       valid_lft forever preferred_lft forever
$ ip -4 addr show virbr0
4: virbr0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN group default qlen 1000
    inet 192.168.122.1/24 brd 192.168.122.255 scope global virbr0
       valid_lft forever preferred_lft forever
```

So the 124 we have here now probably makes sense.  But some kind of docs explaining the different subnets would help.

Reply:
> But some kind of docs explaining the different subnets would help.

Ah, the 124 here should be right.  Maybe just mention `ipRange` again down here?  We already mention it [at the top][1].

[1]: https://github.com/openshift/installer/blob/00862d41c0d5cdb0f92416161651e47ce2481b40/Documentation/dev/libvirt-howto.md#11-pick-a-name-and-ip-range

Reply:
Meanwhile I've done exactly that :-D updated